airflow/models/baseoperator.py

```diff
@@ -1354,7 +1354,7 @@ def render_template_fields(
         context: Context,
         jinja_env: jinja2.Environment | None = None,
     ) -> None:
-        """Template all attributes listed in *self.template_fields*.
+        """Template all attributes listed in *self.template_fields* except for callable.
```

Comment:
I think we can just change this to say like **Render** all attributes listed in *self.template_fields*? Since we do handle callables in this function (by way of `_do_render_template_fields`, they’re just not templated.